### ISSUE TITLE
[Bug]: win.loadUrl() not triggered in a windows application in .scr

### ISSUE THREAD

**Issue by osan15** [electron/electron#28275]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

12.0.1

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Pro 10.0.1942

### What arch are you using?

x64

### Expected Behavior

When I use the api win.loadUrl()
the promise must be resolved or fail. It should not get stuck.

### Actual Behavior

Under windows it is possible to rename the executable of the application in ".scr" to be able to use the electron application as a screensaver.  When I launch my application in simple executable (.exe) it behaves correctly and I receive all the events.
However when the application is launched in "screen saver" mode there is an API that does not respond.

const url = 'file://...';
const loadUrlPromise = await win.loadURL(url); 
await loadUrlPromise.catch(err => {
    console.error('load url failed', err)
     throw err;
 });
// the loadURL promise is never resolved to success or error

I tried to add listening for other events :
win.webContents.on('dom-ready')
win.webContents.on('did-start-loading')
win.webContents.on('did-stop-loading')
win.webContents.on('did-finish-load')
win.webContents.on('did-fail-load')
process.on('uncaughtException')

No event is triggered.

I think the problem is present on Electron 12/11, I will test on other versions but before the api worked.

To test it, there is already an example application to make screensavers for windows.
[github](https://github.com/RandScullard/photo-screen-saver)

**Comment by nornagon** [comment]
Thanks for reporting this and helping to make Electron better!

Because of time constraints, triaging code with third-party dependencies is usually not feasible for a small team like Electron's.

Would it be possible for you to make a standalone testcase with only the code necessary to reproduce the issue? For example, [Electron Fiddle](https://www.electronjs.org/fiddle) is a great tool for making small test cases and makes it easy to publish your test case to a [gist](https://gist.github.com) that Electron maintainers can use.

Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the `blocked/need-repro` label for this reason. After you make a test case, please link to it in a followup comment.

Thanks in advance! Your help is appreciated.

**Comment by osan15** [comment]
Hello,

I just made a github containing the simplest possible example to reproduce the bug.
I couldn't use "Electron Fiddle" because I need to generate a precise file to use the application as a screensaver under Windows.
I documented the README of the project to indicate the procedure to reproduce the bug.

I have also tested other versions of electron:
- platform=win32 arch=x64 electron=8.5.5 => ok
- platform=win32 arch=x64 electron=9.4.4 => ok
- platform=win32 arch=x64 electron=10.4.1 => nok

It seems that the problem exists since version 10 of electron.

Here is the github link :
[https://github.com/osan15/electron-issue-load-url](https://github.com/osan15/electron-issue-load-url)

Thank you in advance
